open-source/overview.mdx

```diff
@@ -15,7 +15,12 @@ file. For all examples we're gonna assume that TLS certificates can be found in
 components in `/var/edgee/wasm`.
 
 <CodeGroup>
-```console docker CLI
+```shell Installer script
+curl --proto '=https' --tlsv1.2 -sSf https://install.edgee.cloud | sh
```

Comment:
Why specify proto and tls version? Could we call `https://install.edgee.cloud` without?

Reply:
These flags are put here to ensure the HTTP request is made securely with minimal supply-chain attack possibilities, it ensures no redirection to insecure website is made and that no insecure TLS version is used, in case of a compromised DNS or something like that
Of course it's not 100% secure but since the whole "curl pipe sh" is not secure already, it reduces the attack range at least.

Reply:
Mmhhh, I'm not sure I understand... 
IMHO, forcing the protocol and the tls version doesn't solve a man-in-the-middle attack. I would prefer to have a simple instruction `curl https://install.edgee.cloud | sh`. It's easy to understand, efficient, straight to the point. 
And developers are free to overload their curl instruction if they wish.

Reply:
Got it, thanks for the feedback! I will fix that and update the PR :+1:

I actually got the `curl` command with flags from https://rustup.rs :)